Automattic/studio
# Local Environment: Fix resolve for external hosts

Fixes https://github.com/Automattic/dotcom-forge/issues/6808
Fixes https://github.com/Automattic/dotcom-forge/issues/6738

## Proposed Changes

This PR adds an extra `mu-plugin` to properly resolve domains for embedded urls.

## Testing Instructions
### Test 1
1. Create a new site
2. Open `wp-admin`
3. Create a new page
4. Add a new video block
5. Try to embed a Youtube video. You should see something like below:
![2024-05-01T11:17:58,413230781+03:00](https://github.com/Automattic/studio/assets/497103/40dfb021-6dd6-4206-8f38-66c127add49f)
6. Try to also embed a SoundCloud page.
### Test 2
1. Create a new site
2. Open `wp-admin`
3. Export a WPCOM site of yours. ( Ensure that it has media )
4. Navigate to `Tools -> Import`. 
5. Import the exported XML file. Ensure that you have "ownload and import file attachments" selected.
6. Ensure that after running import you don't see any errors.



## Pre-merge Checklist
